CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# How to contribute
+
+awstpr is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers and other resources to make getting your contribution into awstpr easier.
+
+# Email and chat
+
+- Email: [giantswarm](https://groups.google.com/forum/#!forum/giantswarm)
+- IRC: #[giantswarm](irc://irc.freenode.org:6667/#giantswarm) IRC channel on freenode.org
+
+## Getting started
+
+- Fork the repository on GitHub
+- Read the [README.md](https://github.com/giantswarm/example-opensource-repo/blob/master/README.md) for build instructions
```

Comment:
This link should go to `https://github.com/giantswarm/awstpr/blob/master/README.md`, not to the example